src/foam/demos/net/nap/web/model/Messageboard.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * @license
+ * Copyright 2018 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.demos.net.nap.web.model',
+  name: 'Messageboard',
+
+  tableColumns: [
+    'mark', 'title', 'createdDate', 'creator', 'attachment', 'hits'
+  ],
+
+  imports: [
+    'blobService',
+    'messageboardDAO',
+    'user'
+  ],
+
+  javaImports: [ 'java.util.Date' ],
+
+  requires: [
+    'foam.blob.BlobBlob',
+    'foam.nanos.fs.File',
+    'foam.u2.dialog.NotificationMessage'
+  ],
+
+  properties: [
+    {
+      class: 'Long',
+      name: 'id'
+    },
+    {
+      class: 'Boolean',
+      name: 'mark',
```

Comment:
Choose a more meaningful property name or add documentation: so people know what this property is for.